scipy/stats/_morestats.py

```diff
@@ -1579,10 +1579,17 @@ def yeojohnson_llf(lmb, data):
         return np.nan
 
     trans = _yeojohnson_transform(data, lmb)
+    trans_var = trans.var(axis=0)
+    loglike = np.empty_like(trans_var)
 
-    loglike = -n_samples / 2 * np.log(trans.var(axis=0))
-    loglike += (lmb - 1) * (np.sign(data) * np.log(np.abs(data) + 1)).sum(axis=0)
+    # Avoid RuntimeWarning raised by np.log when the variance is too low
+    tiny_variance = trans_var < np.finfo(trans_var).tiny
```

Comment:
Oh, I didn't mean you had to change these - I was really just curious how you found those constants hidden away in `_constants`. 
But this is probably better in case `trans_var` is not `float64`.